src/components/ProfitAndLossDownloadButton/ProfitAndLossDownloadButton.tsx

```diff
@@ -15,16 +17,21 @@ export interface PnLDownloadButtonStringOverrides {
 
 export interface ProfitAndLossDownloadButtonProps {
   stringOverrides?: PnLDownloadButtonStringOverrides
+  comparisonConfig?: ProfitAndLossCompareOptionsProps
```

Comment:
Perhaps having boolean field instead would be better? Then we don't have to provide whole `ProfitAndLossCompareOptionsProps` to enable this mode